cleaner.yaml

```diff
@@ -60,6 +60,8 @@ spec:
             env:
             - name: MAX_NAMESPACE_INACTIVE_HOURS
               value: "24"
+            - name: NS_WHITELIST
+              value: "aether-master, aether-demo"
```

Comment:
I am working on a way to add namespace specific variable so that we can use variable to whitelist namespace (e.g. set allowCleanup to false for master and demo). Not sure when will that be ready so I think it's fine to whitelist with this variable now

Reply:
neat.